stdlib/email/message.pyi

```diff
@@ -12,7 +12,7 @@ __all__ = ["Message", "EmailMessage"]
 
 _T = TypeVar("_T")
 
-_PayloadType: TypeAlias = list[Message] | str | bytes
+_PayloadType: TypeAlias = list[Message] | str | bytes | bytearray
```

Comment:
This change means that `get_payload()` can suddenly return `bytearray`, which contradicts the documentation. `set_payload()` is accepting a `bytearray`, though (and converting it to a `str`).

Reply:
`get_payload()` returns Any in our stub, `_PayloadType` is only mentioned in a comment. I think the comment is still close enough, we can revisit it if we ever change the Any.